internal/runners/initialize/init.go

```diff
@@ -158,8 +168,14 @@ func (r *Initialize) Run(params *RunParams) error {
 		return locale.WrapError(err, "err_init_commit", "Could not create initial commit")
 	}
 
-	if err := proj.SetCommit(commitID.String()); err != nil {
-		return locale.WrapError(err, "err_init_setcommit", "Could not store commit to project file")
+	if err := localcommit.Set(proj.Dir(), commitID.String()); err != nil {
+		return errs.Wrap(err, "Unable to create local commit file")
+	}
+	if emptyDir || fileutils.DirExists(filepath.Join(path, ".git")) {
```

Comment:
The AC says "We automatically add the .activestate/commit file to the .gitignore if the current project has a .git directory, **OR if the target directory is empty (eg. fresh checkout)**." (emphasis mine)

I cannot add that conditional logic to `localcommit.AddGitIgnore()` because it will not add it in the empty directory case (at the time of calling, it will have activestate.yaml in it).

If you'd like to change the AC to only add .gitignore if .git/ directory exists, then I can make the conditional inside `localcommit.AddGitIgnore()`.

Reply:
I think that AC still stands. I'd argue you could still assert "empty" by seeing if the dir ONLY contains the activestate.yaml, but this is a nit so entirely up to you on how/if you handle this feedback.